### PR TITLE
Fetch address  case 1124

### PR DESCRIPTION
**Pull request by mlipardo**
WIP

### CODE REVIEW

**Comment by SashaKyz** [comment]
Can one of the admins verify this patch?

File: spec/lib/addresses/geo_repository_spec.rb [hunk 1]
```diff
@@ -0,0 +1,58 @@
+# frozen_string_literal: true
+
+require 'spec_helper'
+require 'addresses/address'
+require 'json'
```
Comment (edgibbs) [inline spec/lib/addresses/geo_repository_spec.rb]: I think you can remove these requires for addresses/address and json

File: spec/lib/addresses/us_states_spec.rb [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+module Addresses
+  describe UsStates do
+    context 'state code exists' do
+      it 'looks up a US state code' do
+        expect(UsStates.new.us_state(1828)).to eq 'CA'
+        expect(UsStates.new.us_state(1823)).to eq 'AK'
+      end
+    end
+
+    context 'state code does not exists' do
```
Comment (edgibbs) [inline spec/lib/addresses/us_states_spec.rb]: tiny typo exist versus exists

File: spec/lib/placements/address_map_service_spec.rb [hunk 3]
```diff
@@ -0,0 +1,114 @@
...
+                    Clients::Client.new(identifier: 'starlord',
+                                        address: Addresses::Address.new(street_name: 'Sesame st',
+                                                                        state_code: 1823,
+                                                                        state: 'AK',
+                                                                        latitude: 1.1,
+                                                                        longitude: 2.2))]
+        end
+      end
+
+      context 'with child address and one relationship no address ' do
```
Comment (edgibbs) [inline spec/lib/placements/address_map_service_spec.rb]: maybe change the description to be 'with child and one relationship but no addresses'

**Comment by mlipardo** [comment]
retest this please.

**Pull request merged**